mico-admin/src/app/api/api.service.ts

```diff
@@ -83,11 +84,13 @@ export class ApiService {
         const stream = this.getStreamSource<ApiModelMap>(resource, () => new AsyncSubject<Readonly<ApiModelMap>>());
 
         // TODO replace URL with a generic path
-        this.rest.get<{ definitions: ApiModelMap, [prop: string]: any }>('http://localhost:8080/v2/api-docs').subscribe(val => {
+        if (!stream.isStopped) {
+            this.rest.get<{ definitions: ApiModelMap, [prop: string]: any }>('http://localhost:8080/v2/api-docs').subscribe(val => {
```

Comment:
remove host part and make this path relative. The api base service knows how to resolve the relative url.